### DISCUSSION TITLE
Repositories put in a list to disappear from the starred list

### DISCUSSION THREAD

**Discussion by OussamaRomdhane** [community/community#8865] [Other Feature Feedback, Questions, & Ideas]
Or alternatively have a toggle that will allow that, I think the whole idea behind lists is to avoid clutter, so it would be nice to see the effect progressively as we rearrange starred repositories into lists.

**Comment by brianlovin** [comment]
Thanks for the suggestion — we've had a few requests for this functionality, so I'll add your +1 to our tracking issue!
